src/ControlSystem/ControlErrors/Size.hpp

```diff
@@ -404,7 +408,8 @@ struct Size : tt::ConformsTo<protocols::ControlError> {
             ? std::optional<double>(control_error_delta_r -
                                     delta_r_drift_outward_options_.value()
                                         .outward_drift_velocity -
-                                    (lambda_00 + horizon_00 -
+                                    (lambda_00 +
+                                     spherepack_factor * horizon_00 -
```

Comment:
Question: how severe is this bug? what impact did it have on simulations?

Reply:
None. In all of my runs I've disabled the `delta_r_drift_inward` option since we haven't needed it.

Reply:
Oh ok good 👍